EnglishFilter/sections/foreign.txt

```diff
@@ -3527,6 +3527,13 @@ tgd.kr###header-top-notice
 @@||hjkhgyss.tistory.com^$generichide
 !+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_safari, ext_ublock)
 @@||daumcdn.net/*#ad&adv_keywords*impopup$domain=hjkhgyss.tistory.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/33603
+!+ PLATFORM(ios, ext_safari, ext_android_cb)
+toonkor.page##.main_top_bann
+!+ PLATFORM(ios, ext_safari, ext_android_cb)
+toonkor.page##.top_banner
```

Comment:
This and the next one hides little left-over space on Windows too, so I guess better to have all rules available to every platform.

Reply:
Ok I deleted platform for that left-over space.